app/models/component/manifest/app_container.rb

```diff
@@ -37,6 +37,10 @@ def environment_settings
         env << { name: '_JAVA_OPTIONS', value: java_options.to_s }.with_indifferent_access
         env << { name: '_BUSBAR_BUILD_TIME', value: timestamp.to_s }.with_indifferent_access
         env << { name: 'PORT', value: app_port.to_s }.with_indifferent_access
+        env << { name: 'DD_AGENT_HOST', valueFrom: { fieldRef: { fieldPath: 'status.hostIP'}}}.with_indifferent_access
```

Comment:
There's a missing space before each of the closing brackets on this line.

Reply:
Maybe I could add just to the more inward one?